File: julia/write_benchmarks.jl
function write_benchmarks(io::IO, bench::AbstractDict{String,<:AbstractDict})
    println(io, "Benchmark,File,Time(s)")
    for (b, d) in bench
        for (f, t) in d
            println(io, b, ',', f, ',', t)
        end
    end
end
function write_benchmarks(filename::AbstractString, bench)
    open(filename, "w") do io
        write_benchmarks(io, bench)
    end
end

File: matlab/write_benchmarks.m
function write_benchmarks(filename, bench)
    fid = fopen(filename, 'w');
    fprintf(fid, 'Benchmark,File,Time(s)\n');
    bs = fieldnames(bench);
    for i = 1:length(bs)
        b = bs{i};
        benchtask = bench.(b);
        fs = fieldnames(benchtask);
        for j = 1:length(fs)
            f = fs{j};
            fprintf(fid, '%s,%s,%g\n', b, f, benchtask.(f));
        end
    end
    fclose(fid);
end
